release-notes.md

```diff
@@ -0,0 +1,55 @@
+# Release v0.7.1
+
+## New features
+* Add [alert manager](prometheus/doc/alert-manager.md), defined [several rules](https://github.com/Microsoft/pai/tree/pai-0.7.y/prometheus/prometheus-alert) for alert. Operator will receive email notification if cluster has some problem.
```

Comment:
do we have an "**Operator**" role in PAI? 

**I tuned the message a little bit:**
Administrators can receive email notifications on cluster problems after set up the new supported "Alert Manager". Please read more about how to set up [Alert Manager](prometheus/doc/alert-manager.md) and the notification [Rules](https://github.com/Microsoft/pai/tree/pai-0.7.y/prometheus/prometheus-alert).